DEVELOPERS.md

```diff
@@ -1,210 +1,210 @@
 # Developing Ergo
-
+ 
 * [Development Setup][developers.setup]
 * [Coding Rules][developers.rules]
 * [Commit Message Guidelines][developers.commits]
 * [Writing Documentation][developers.documentation]
-
+ 
 ## Overview
-
+ 
 This document describes how to set up your development environment to build and test Ergo, and explains the basic mechanics of using `git`, `node`, `lerna`.
-
+ 
 The core of the Ergo compiler is extracted from a formal specification in Coq to JavaScript.
-
+ 
 The code is located in the following directories:
-
+ 
 * in `mechanization` the Coq code \(includes the abstract syntax tree, intermediate languages, and compiler to JavaScript\)
 * in `extraction` support code in OCaml \(includes the parser\)
 * in `packages` the JavaScript packages, containing the Ergo compiler API and command-line tools
-
+ 
 ## <a name="setup"> Development Setup
-
+ 
 ### Installing Dependencies
-
+ 
 Before you can build Ergo, you must install and configure the following dependencies on your machine:
-
-* [Git](http://git-scm.com/): The \[Github Guide to Installing Git\]\[git-setup\] is a good source of information.
+ 
+* [Git](http://git-scm.com/): The [Github Guide to Installing Git][git-setup] is a good source of information.
 * [Node.js v8.x \(LTS\)](http://nodejs.org): We use Node to generate the documentation, run a development web server, run tests, and generate distributable files. Depending on your system, you can install Node either from source or as a pre-packaged bundle.
-
-We recommend using [nvm](https://github.com/creationix/nvm) \(or [nvm-windows](https://github.com/coreybutler/nvm-windows)\) to manage and install Node.js, which makes it easy to change the version of Node.js per project.
-
-* [Lerna](https://lerna.js.org/): We use lerna to handle multiple npm packages in the Ergo repository. To install:
-
+ 
+We recommend using [nvm](https://github.com/creationix/nvm) (or [nvm-windows](https://github.com/coreybutler/nvm-windows)) to manage and install Node.js, which makes it easy to change the version of Node.js per project.
+ 
+* [Lerna](https://lernajs.io): We use lerna to handle multiple npm packages in the Ergo repository. To install:
```

Comment:
This is the wrong link for Lerna. `https://lerna.js.org/` above is the right one.